README.md

```diff
@@ -1,47 +1,39 @@
-# Sourcegraph projects and content repository
+# Sourcegraph about.sourcegraph.com website
 
-> NOTE: If you're writing a blog post for a release, use the [release blog post template](blogposts/RELEASE_BLOG_POST_TEMPLATE.md) 
+Uses Gatsby for rendering a static site that is built and deployed with Netlify.
 
-We're trying something new with this repository, capturing current projects and blog posts as Open Source code in Markdown using embedded YAML for metadata (front matter).
+## Requirements
 
-The publishing of these blog posts is automatic, using the value of the `published` front matter key.
+Gatsby requires Node.js and we recommend using [yarn](https://yarnpkg.com/en/) for installing packages.
 
-## Why we're doing this
+## Developing locally
 
-Sourcegraph is committed to Open Source and developing in the open as much as possible. While we currently do our iteration planning in Google docs, we can't use the tools we know and love as Developers. We want to be in a terminal or editor writing code, not a web browser writing a document.
+In a terminal, change into the `website` directory, then run:
 
-We also want more transparency in what we're working on (projects) and what we're writing about (blogs), so we can get as much feedback as possible during the creation and review process outside of just the Sourcegraph team.
+```shell
+yarn install
+```
 
-## What we hope to get out of this
+Then to serve the site, you have a two options.
 
-We build tools to help developers write better quality code, and we want those same developers to tell us what would make them more productive and successful.
+To serve the static (built) version of the site (good for final review):
 
-We want feedback while we're working on features and creating content to ensure everything we release and publish is incredibly high quality and meets the needs of our developer audience.
+```shell
+yarn serve
+```
 
-## For non-Sourcegraph Team members. How you can contribute
+To serve the site using the development server with auto-compiling and page reloading:
 
-- Watch this repository for updates to see what's happening. Share your thoughts with us here on [on Twitter](http://twitter.com/srcgraph).
-- Create issues for features or content you want to see.
-- Create issues telling us how we can improve our project and content sharing process.
+```shell
```

Comment:
Indent these two code snippets (line 21-23 and 27-29) to be nested under the numbers.

P.S. Super sorry about all the suggested changes - first time using it and wasn't sure how it would work with a bunch of simple formatting. In hindsight it was not the best form of suggestions...